platforms/express.ts

```diff
@@ -43,7 +44,12 @@ export function serve<TInitialPayload = unknown>(
 
     // create handler
     const { handler: serveHandler } = serveBase<TInitialPayload>(
-      (workflowContext) => routeFunction(workflowContext),
+      routeFunction,
+      {
+        sdk: SDK_TELEMETRY,
+        platform: "express",
+        runtime: `node@${process.version}`,
```

Comment:
How can we make sure it's node? It can also be bun etc... can we somehow detect that?

Reply:
This applies to all the configs

Reply:
https://bun.sh/guides/util/detect-bun#:~:text=The%20recommended%20way%20to%20conditionally,being%20executed%20in%20a%20browser.